src/resolvers/registries/npm-resolver.js

```diff
@@ -180,6 +181,18 @@ export default class NpmResolver extends RegistryResolver {
     // lockfile
     const shrunk = this.request.getLocked('tarball');
     if (shrunk) {
+      if (this.config.packBuiltPackages && shrunk.prebuiltVariants && shrunk._remote) {
+        const prebuiltVariants = shrunk.prebuiltVariants;
+        const prebuiltName = getPlatformSpecificPackageFilename(shrunk);
+        const offlineMirrorPath = this.config.getOfflineMirrorPath();
+        if (prebuiltVariants[prebuiltName] && offlineMirrorPath) {
+          const filename = path.join(offlineMirrorPath, 'prebuilt', prebuiltName + '.tgz');
+          if (shrunk._remote && (await fs.exists(filename))) {
```

Comment:
Fwiw the Node documentation recommends using `fs.existsSync` instead of `fs.exists` to prevent race condition as much as possible (`fs.exists` is actually deprecated).

Reply:
We have 59 places where this wrapper is used.
If we migrate away from it we should do it in a separate PR, right?